hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/Route.java

```diff
@@ -131,6 +131,7 @@ public void configure() throws Exception {
     // Ideally this happens in the Constructor but @PropertyInject happens after the constructor so we call it from the route itself
     // To call it from the constructor we could move property loading into MainMethod and pass the properties into the Constructor
     public void init() throws Exception {
+    	clientId = (System.getenv("MOH_HNCLIENT_ID")!=null) ? System.getenv("MOH_HNCLIENT_ID") : clientId;
```

Comment:
MOH_CLIENT_ID should be a constant...especially since it's used twice. I'd probably put it in a local variable as well to avoid calling getenv() twice but that's mostly preference. Please add white space to != null check. Also maybe add a comment to explain why this is done...it can be a little confusing to see an injected property overwritten.

Reply:
updated the code for single call and added comments